docs/v.list

```diff
@@ -4,60 +4,55 @@
+    <var name="kotlinReleaseDate" value="October 10, 2024" type="string"/>
+    <var name="kotlinLatestUrl" value="https://github.com/JetBrains/kotlin/releases/tag/v2.0.21" type="string"/>
 
+    <var name="languageVersion" value="2.0" type="string"/>
+    <var name="apiVersion" value="2.0" type="string"/>
+
+    <!-- Kotlin EAP -->
     <var name="kotlinEapVersion" value="2.1.0-Beta2" type="string"/>
+    <var name="kotlinEapReleaseDate" value="October 14, 2024" type="string"/>
 
-    <var name="kotlinLatestUrl" value="https://github.com/JetBrains/kotlin/releases/tag/v2.0.21" type="string"/>
+    <!-- Libraries and Frameworks -->
+    <var name="coroutinesVersion" value="1.9.0" type="string"/>
+    <var name="coroutinesEapVersion" value="1.9.0-RC.2" type="string"/>
 
-    <var name="kotlinReleaseDate" value="October 10, 2024" type="string"/>
+    <var name="serializationVersion" value="1.7.3" type="string"/>
 
-    <var name="kotlinEapReleaseDate" value="October 14, 2024" type="string"/>
+    <var name="dateTimeVersion" value="0.6.1" type="string"/>
 
-    <var name="coroutinesVersion" value="1.8.1" type="string"/>
+    <var name="ktorVersion" value="3.0.0" type="string"/>
 
-    <var name="coroutinesEapVersion" value="1.9.0-RC.2" type="string"/>
+    <var name="sqlDelightVersion" value="2.0.2" type="string"/>
 
+    <!-- Gradle, KGP, AGP -->
     <var name="minGradleVersion" value="6.8.3" type="string"/>
-
     <var name="maxGradleVersion" value="8.8" type="string"/>
 
     <var name="minAndroidGradleVersion" value="7.1.3" type="string"/>
-
     <var name="maxAndroidGradleVersion" value="8.5" type="string"/>
 
     <var name="gradleVersion" value="8.5" type="string"/>
-
     <var name="gradleLanguageVersion" value="KOTLIN_2_0" type="string"/>
-
     <var name="gradleApiVersion" value="KOTLIN_2_0" type="string"/>
 
-    <var name="serializationVersion" value="1.7.1" type="string"/>
+    <!-- KSP -->
+    <var name="kspSupportedKotlinVersion" value="2.0.21" type="string"/>
+    <var name="kspVersion" value="1.0.25" type="string"/>
 
+    <!-- Other -->
     <var name="defaultJvmTargetVersion" value="1.8" type="string"/>
 
     <var name="jvmLTSVersionSupportedByKotlin" value="17" type="string"/>
 
     <var name="webpackMajorVersion" value="5" type="string"/>
-
     <var name="webpackPreviousMajorVersion" value="4" type="string"/>
 
-    <var name="sqlDelightVersion" value="2.0.2" type="string"/>
-
-    <var name="kspSupportedKotlinVersion" value="2.0.20" type="string"/>
-
-    <var name="kspVersion" value="1.0.24" type="string"/>
-
-    <var name="ktorVersion" value="2.3.12" type="string"/>
-
-    <var name="dateTimeVersion" value="0.6.0" type="string"/>
-
     <var name="foojayResolver" value="0.5.0" type="string"/>
 
-    <var name="languageVersion" value="2.0" type="string"/>
-
-    <var name="apiVersion" value="2.0" type="string"/>
-
     <var name="lincheckVersion" value="2.34" type="string"/>
```

Comment:
I think Lincheck should also be in the Libraries and Frameworks section

Reply:
I've moved, thanks!